src/Model/ExternalArticle.php

```diff
@@ -39,4 +39,19 @@ public function getUri(): string
     {
         return $this->uri;
     }
+
+    public function getId(): string
+    {
+        return $this->uri;
+    }
+
+    public function getType(): string
+    {
+        return 'external-article';
+    }
+
+    public function getTitle(): string
+    {
+        return $this->getArticleTitle();
```

Comment:
I'd just rename `getArticleTitle()` (the object is called `ExternalArticle`, so it's clearly the title of the article).